src/init.jl

```diff
@@ -38,6 +38,8 @@ function init()
     JavaCall.addOpts("-Xmx1024M")
     try
         JavaCall.init()
+    catch
+        # ?
```

Comment:
Yeah, maybe get rid of the try/catch here. If `JavaCall.init()` fails, we should probably bail out?

Reply:
It might have been related to loading code in REPL a long ago, but now I don't see any direct usage, so let's remove it, at least for now.